lib/transforms/integration/set-value.js

```diff
@@ -30,30 +30,35 @@ function createBlurExpression(j, selector) {
 }
 
 /**
- * Check if `node` is a `this.$(selector).val(someValue)` expression
+ * Check if `node` is a `this.$(selector).val(someValue)` expression not in an arrow function
  *
  * @param j
  * @param node
  * @returns {*|boolean}
  */
-function isJQueryExpression(j, node) {
+function isJQueryExpression(j, path) {
+  let node = path.node;
+  let grandGrandParent = path.parent.parent.parent.node;
```

Comment:
Can we always assume that there is always a grand grand parent? Might throw if some `.parent` is `undefined`?